ui/src/shared/components/TableGraph.js

```diff
@@ -25,21 +27,49 @@ class TableGraph extends Component {
       unzippedData: [[]],
       hoveredColumnIndex: NULL_COLUMN_INDEX,
       hoveredRowIndex: NULL_ROW_INDEX,
-      sortByColumnIndex: -1,
+      sortByColumnIndex: NULL_COLUMN_INDEX,
+      clicktoSortFieldIndex: NULL_COLUMN_INDEX,
+      clicktoSortDirection: DEFAULT_SORT,
     }
   }
 
   componentWillReceiveProps(nextProps) {
-    const {data, unzippedData} = timeSeriesToTableGraph(nextProps.data)
-
+    const {data} = timeSeriesToTableGraph(nextProps.data)
```

Comment:
are you getting unzippedData from timeSeriesToTableGraph anymore? you can probably stop it from doing that calculation there so as not to duplicate work

Reply:
good point thanks.